Makefile

```diff
@@ -1,13 +1,26 @@
-DOCKER_CI_IMAGE = go-ceph-ci
-CONTAINER_CMD := docker
+CI_IMAGE_NAME = go-ceph-ci
+CONTAINER_CMD ?=
 CONTAINER_OPTS := --security-opt $(shell grep -q selinux /sys/kernel/security/lsm && echo "label=disable" || echo "apparmor:unconfined")
 CONTAINER_CONFIG_DIR := testing/containers/ceph
-VOLUME_FLAGS := 
+VOLUME_FLAGS :=
 CEPH_VERSION := nautilus
 RESULTS_DIR :=
 CHECK_GOFMT_FLAGS := -e -s -l
 IMPLEMENTS_OPTS :=
 
+ifeq ($(CONTAINER_CMD),)
+	CONTAINER_CMD:=$(shell docker version >/dev/null 2>&1 && echo docker)
+endif
+ifeq ($(CONTAINER_CMD),)
```

Comment:
I would prefer to use podman, and not docker. The order of these checks can be changed :wink: